03_reverseString/README.md

```diff
@@ -6,5 +6,8 @@ Pretty simple, write a function called `reverseString` that returns its input, r
 reverseString('hello there') // returns 'ereht olleh'
 ```
 
+You will notice in this exercise that there are multiple tests (see in file reverseString.spec.js). Only the first test is currently enabled. So after making sure that this first one passes, enable the others one by one by deleting the .skip from the test.skip() function.
```

Comment:
> You will notice in this exercise that there are multiple tests (see in file reverseString.spec.js). Only the first test is currently enabled. So after making sure that this first one passes, enable the others one by one by deleting the .skip from the test.skip() function.

A couple of changes please. Can we change `see in file reverseString.spec.js` to something a bit more readable such as "Review the reverseString.spec.js file for details".

`Only the first test is currently enabled. So after making`. Can we either make the full stop a comma and then downcase `so` or keep the full stop but remove `So` and just start the next sentence with `After`